src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQuerySource.java

```diff
@@ -82,7 +71,7 @@
   private Configuration configuration;
   private final BigQueryAvroToStructuredTransformer transformer = new BigQueryAvroToStructuredTransformer();
   // UUID for the run. Will be used as bucket name if bucket is not provided.
-  private UUID uuid;
+  private String runId;
```

Comment:
We are already using a run id term (ProgramRunId) and it looks like this guy is something else. Please use a different name

Reply:
This is an identifier for a given execution. Each source has it's own "runId" that is used to create a new path to store objects related to this source.

Reply:
Well, it's not a key for execution, you can't retrieve an execution using it. It would be no problems if we did not have already io.cdap.cdap.common.app.RunIds widely used, but now those two will be confused and if someone try to retrieve a run using this id he/she would fail.

Reply:
I see your point now.

Reply:
I have renamed this variable to something that is less confusing.